blog_style.css

```diff
@@ -56,4 +56,10 @@ header h3 {
 		width: 90%;
 		margin: 100px 5%;
 	}
+}
+
+@media (max-width: 480px) {
+	header a:nth-child(1) {
```

Comment:
@fcvernon , generally, I like to add 2 spaces rather than 4 for my tabs. Otherwise, this looks great.